.rubocop.yml

```diff
@@ -1,33 +1,23 @@
-inherit_gem:
-  rubocop-rootstrap:
-    - config/rails_edge.yml
+inherit_from:
+  - .rubocop-rs-config.yml
+  - .rubocop-excludes.yml
```

Comment:
Yes, I know it's part of the rs config but I thought could be better if we have all the exclusions in a specific file because it is easier to visualize and you can see all the places where you are excluding a cop. In the other file, there are too many rules and information. But it was just my idea, if you don't like it I can remove it.